olp-cpp-sdk-core/src/http/curl/NetworkCurl.h

```diff
@@ -152,9 +151,6 @@ class NetworkCurl : public olp::http::Network,
   /// Number of CURL easy handles that are always opened.
   static constexpr int kStaticHandleCount = 8;
 
-  /// Maximum allowed number of CURL easy handles.
-  static constexpr int kTotalHandleCount = 32;
```

Comment:
Please, fix CreateDefaultNetworkRequestHandler default param value, because it isn't 32

Reply:
Before this PR, there was 32 handles if CreateDefaultNetworkRequestHandler without explicit parameter was used. Now curl network respect parameter from CreateDefaultNetworkRequestHandler and default behavior become 30 handles. I'd rather, you preserve previous defaults.